File: Data Structures For Interview/Java/Array Programs/Frequency.java
class Frequency{

    static void calculateFrequency(int arr[]){

        int array[]=new int[arr.length];
        
        for(int i=0;i<array.length;i++){
            array[i]=0;
        }

        for(int i=0;i<arr.length;i++){
            int element=arr[i];
            int count=1;
            int flag=0;

            for(int k=0;k<array.length;k++){
                if(array[k]==element){
                    flag=1;
                    break;
                }
            }

            if(flag==0){
                for(int j=i+1;j<arr.length;j++){
                    if(element==arr[j]){
                        count++;
                    }
                }
                System.out.println("Frequency of "+element+" is :- "+count);
                
            }
            count=0;
            array[i]=element;



           
        }
    }
    public static void main(String s[]){

        int arr[]={10,10,10,10,10,10,20,20,20,20,20,30,30,30,30,30,40,40,40,40,50,50,50,60,60};

        calculateFrequency(arr);

    }
}

File: Data Structures For Interview/JavaScript/Array Programs/Frequency.js
function frequency(arr){

    let array=[]

    for(let i=0;i<arr.length;i++){
        
        let element=arr[i];
        let count=1;
        let flag=0;

        for(let k=0;k<array.length;k++){
            if(array[k]==element){
                flag=1;
                break;
            }
        }

        if(flag==0){
            for(let j=i+1;j<arr.length;j++){
                if(arr[j]==element){
                    count++;
                }
            }
    
            console.log("Frequency of "+element+" is :- "+count);
        }

       array[i]=element;
        
    }
}

let arr=[1,2,3,4,5,6,7,8,9,10,3,3,4,4,5,5,6,6];
frequency(arr);